Sources/Files.swift

```diff
@@ -204,7 +204,7 @@ public final class Storage<LocationType: Location> {
     private let fileManager: FileManager
 
     fileprivate init(path: String, fileManager: FileManager) throws {
-        self.path = path
+        self.path = path.replacingOccurrences(of: "file://", with: "")
```

Comment:
A path with the `file://` prefix is not valid, so passing in that kind of path is a programmer error, and not something I think Files should handle like this. That being said, I'd be totally open to adding an `init(url: URL)` convenience initializer that takes care of correctly translating a `file://` URL into a path.